backend/src/database/state-migrations/10-rename-delete-transfer-action.ts

```diff
@@ -0,0 +1,27 @@
+import type { Action } from 'digital-fuesim-manv-shared';
+import type { DeleteTransferAction } from 'digital-fuesim-manv-shared/dist/store/action-reducers/transfer';
+import type { Migration } from './migrations';
+
+export const renameDeleteTransferAction10: Migration = {
+    actions: (_initialState, actions) => {
+        for (let i = 0; i < actions.length; i++) {
+            const action = actions[i] as Action | null;
+            if (action?.type === '[Transfer] delete transfer') {
+                const oldAction = action as unknown as {
+                    elementId: string;
+                    elementType: 'personnel' | 'vehicles';
+                    targetTransferPointId: string;
+                };
+                const newAction: DeleteTransferAction = {
+                    type: '[Transfer] Delete transfer',
+                    elementId: oldAction.elementId,
+                    elementType: oldAction.elementType,
+                    targetTransferPointId: oldAction.targetTransferPointId,
+                };
+
+                actions[i] = newAction;
```

Comment:
I think you can just reset the name of the action without creating a new one.

Reply:
I did not do it, since the type is readonly, but I think simply not casting to Action should do the trick, thank you for the hint.

I'm going to take a look at this tomorrow

Reply:
The actions are actually `Mutable<object>` - sadly such a type doesn't exist.

````ts
        for (const action of actions as (Mutable<Action> | null)[]) {
            if (action?.type === '[Transfer] delete transfer') {
                action.type = '[Transfer] Delete transfer';
            }
        }
````